ws_sdk/app.py

```diff
@@ -727,6 +738,24 @@ def get_products(self,
                                                sort_by=sort_by)
         return products
 
+    def get_product_by_name(self,
+                            name: str) -> dict:
+        products = self.get_products(name=name)
+        for product in products:
+            if product['name'] == name:
+                return product
+        logger.error(f"Product with name: {name} was not found")
+        raise WsSdkServerMissingTokenError(name, ScopeTypes.PRODUCT)
+
+    def get_product_by_token(self,
```

Comment:
check [get_scope_by_token()](https://github.com/whitesource-ps/ws-sdk/blob/6cd8fb95a8197169092d7fb057babbf854ca5b09/ws_sdk/app.py#L463) to be used instead.

Reply:
This is also same deal we want a specific product not a dictionary of an entire scope still feel it is useful

Reply:
Same as [mentioned here](https://github.com/whitesource-ps/ws-sdk/pull/99/files#r830584495)